tests/c_languages/CPPCheckBearTest.py

```diff
@@ -1,37 +1,34 @@
+import os
+
```

Comment:
you can leave this newline, `queue` is a system import ;)

Reply:
okay, will remove that